docs/develop/develop-overview.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Develop Office Add-ins
+description: An introduction to developing Office Add-ins.
+ms.date: 10/28/2019
+localization_priority: Priority
+---
+
+# Develop Office Add-ins
+
+Office Add-ins can extend the UI and functionality of Office applications and interact with content in Office documents. You can use familiar web technologies to create Office Add-ins that extend and interact with Word, Excel, PowerPoint, OneNote, Project, or Outlook, and your solution can run in Office across multiple platforms, including Windows, Mac, iPad, and in a browser. This article provides an introduction to developing Office Add-ins.
+
+> [!TIP]
+> If you haven't already done so, please review [Office Add-ins platform overview](../overview/office-add-ins.md) for information that sets context for the topics covered in this article.
+
+## Core development concepts 
+
+As described in [Office Add-ins platform overview](../overview/office-add-ins.md), an Office Add-in consists of two parts:
+
+- The add-in manifest (an XML file) that defines the settings and capabilities of the add-in.
+
+- The web application that defines the UI and functionality of add-in components such as task panes, content add-ins, and dialog boxes.
+
+The web application can use the Office JavaScript API to interact with content in the Office document where the add-in is running, and can also do other things that web applications typically do, like call external web services, facilitate user authentication, and more.
+
+### Defining an add-in's settings and capabilities
+
+An Office Add-in's manifest (an XML file) defines the settings and capabilities of the add-in. You can configure the manifest to specify things such as:
```

Comment:
Consider: "You can configure" -> "You'll configure". Their only choice is to configure the manifest.